src/PrestaShopBundle/Utils/ZipManager.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace PrestaShopBundle\Utils;
+
+use RecursiveIteratorIterator;
+use RecursiveDirectoryIterator;
+
+class ZipManager
```

Comment:
If we also add a function to defate zips, this class will be used as a dependancy for other classes (i.e ModuleZipManger, which has its specific functionnalities but could have shared code)